### PR TITLE
Basic cloudevent example

### PR DESCRIPTION
**Pull request by subfuzion**

### CODE REVIEW

File: .github/workflows/verify_examples.yml [hunk 1]
```diff
@@ -18,7 +18,9 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        example_directory: ["examples/hello", "examples/json"]
+        example_directory:
+          - examples/hello
+          - examples/raw_cloudevent
```
Comment (kevmoo) [inline .github/workflows/verify_examples.yml:23]: Add back the json example?

File: examples/raw_cloudevent/build.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+targets:
+  $default:
+    builders:
+      source_gen|combining_builder:
+        options:
+          ignore_for_file:
+          - prefer_expression_function_bodies
+
```
Comment (kevmoo) [inline examples/raw_cloudevent/build.yaml:8]: one extra new-line?

Comment (kevmoo) [reply examples/raw_cloudevent/build.yaml:8]: Oh! This is only needed if you're using json_serializable. Can remove!

**Review by kevmoo — CHANGES_REQUESTED** [review]

File: examples/raw_cloudevent/build.yaml (hunk 2 above)
Comment (subfuzion) [reply examples/raw_cloudevent/build.yaml:8]: ah, gotcha, thanks!

File: .github/workflows/verify_examples.yml (hunk 1 above)
Comment (subfuzion) [reply .github/workflows/verify_examples.yml:23]: On it!

File: examples/raw_cloudevent/build.yaml (hunk 2 above)
Comment (subfuzion) [reply examples/raw_cloudevent/build.yaml:8]: This area is still a bit of an arcane art for me ... I've removed `options` section (I think that's what you're saying)..

Comment (subfuzion) [reply examples/raw_cloudevent/build.yaml]: @kevmoo PTAL now, thanks!

Comment (subfuzion) [reply examples/raw_cloudevent/build.yaml]: ruh roh ... lemme see why build check failed first.

File: examples/raw_cloudevent/build.yaml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+targets:
```
Comment (kevmoo) [inline examples/raw_cloudevent/build.yaml:1]: This file isn't needed AT ALL!

It's weird for the JSON example. All about lints. Better to explain over VC

Comment (subfuzion) [reply examples/raw_cloudevent/build.yaml:1]: ah ok, misunderstood, removing. Yeah would love to understand this better, but it's a saturday -- we can chat next week. Removing file now, thanks!

File: examples/raw_cloudevent/build.yaml (hunk 2 above)
Comment (subfuzion) [reply examples/raw_cloudevent/build.yaml]: Ok, fixed now. PTAL when you have a moment, thanks. I will add Cloud Run docs for this after we're able to publish v0.3.0 and remove the pub overrides.

File: examples/raw_cloudevent/build.yaml (hunk 3 above)
Comment (subfuzion) [reply examples/raw_cloudevent/build.yaml]: Removed, PTAL when you have a chance, thanks.

**Review by kevmoo — APPROVED** [review]

**Pull request merged**